src/main/groovy/info/solidsoft/gradle/pitest/PitestPluginExtension.groovy

```diff
@@ -47,6 +47,7 @@ class PitestPluginExtension {
     BigDecimal timeoutFactor
     Integer timeoutConstInMillis
     Integer maxMutationsPerClass
+    String testPlugin
```

Comment:
The new parameters are briefly documented in this file (as it seems to be easier for people to look here than reading the original PIT documentation. You could add some description (including `@since` - targeting 1.3.0).

Reply:
Btw, as in PIT that option was added as the [first one](https://github.com/hcoles/pitest/blob/2923d86d71b03b743a7ea3b1d1d06b3dbf67ca5f/pitest-entry/src/main/java/org/pitest/mutationtest/config/ConfigOption.java#L26) you could place it just above the `reportDir` in that file (to be somehow coherent)